nixos/modules/services/mail/sympa.nix

```diff
@@ -0,0 +1,536 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.sympa;
+  user = cfg.user;
```

Comment:
I'd not `let` the user and group, but that's probably more of a personal opinion.